dev/integration_tests/asset_transformation/integration_test/integration_test.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:asset_transformation/main.dart' as app;
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('able to render svg asset, which should have been compiled at build time', (WidgetTester tester) async {
```

Comment:
are we actually testing this?

Reply:
This was an artifact of an older implementation. I've removed it. Thanks!